.github/actions/setup-sui/action.yaml

```diff
@@ -1,15 +1,19 @@
-name: Install Sui CLI
-description: 'Install Sui CLI'
+name: Setup SUI CLI
+description: 'Setup Sui CLI'
 
 inputs:
   sui-version:
-    description: 'Sui Version'
+    description: 'The version of Sui CLI to install'
     required: true
 
 runs:
   using: 'composite'
 
   steps:
+    - name: Get Sui Version
```

Comment:
is this needed, or is it visible in logs when the action is called with the inputs?